jaggr-core/pom.xml

```diff
@@ -72,7 +72,7 @@
   <build>
     <resources>
       <resource>
-        <directory>..</directory>
+        <directory>${project.parent.basedir}</directory>
```

Comment:
cool.   This is more proper anyway.  The .. was a hack, i didn't realize you could use that variable.